util/expensivequery/expensivequery.go

```diff
@@ -124,8 +124,8 @@ func logExpensiveQuery(costTime time.Duration, info *util.ProcessInfo) {
 	if len(info.User) > 0 {
 		logFields = append(logFields, zap.String("user", info.User))
 	}
-	if len(info.DB) > 0 {
-		logFields = append(logFields, zap.String("database", info.DB))
+	if info.DB != nil && len(info.DB.(string)) > 0 {
```

Comment:
Why change this?

Reply:
I change the type of `DB` and `Info` from `string` to `interface{}`.   If not change this, it get a panic.

Reply:
OK, it seems that we can also process `DB` using the same way as https://github.com/pingcap/tidb/pull/10985/files#r298861952 mentioned.

Reply:
When do something to the table `information_schema.processlist` , it will get a wrong result. eg: `select * from information_schema.processlist where db is null;`